PlayCover/Views/Sidebar Views/IPALibraryView.swift

```diff
@@ -65,11 +36,106 @@ struct IPALibraryView: View {
                     Spacer()
                 }
                 .frame(maxWidth: .infinity, maxHeight: .infinity)
+                .sheet(isPresented: $addSourcePresented) {
+                    AddSourceView(addSourceSheet: $addSourcePresented)
+                        .environmentObject(storeVM)
+                }
+            } else {
+                if #available(macOS 13.0, *) {
```

Comment:
Again question here about the necessity of the `#available`

Reply:
same as previously answered but with the difference of navigation stack and navigation link being used in macOS 13 part that are not available in macOS 12 so we need to provide what behaviour macOS 12 should have